### PR TITLE
Updates to UG on a11y and 2 CC error messages

### PR DESCRIPTION
**Pull request by cpandhi**

### CODE REVIEW

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx [hunk 1]
```diff
@@ -957,6 +962,94 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
               </ol>
             </div>
 
+            <h3 id="a11y_considerations">6.5 Accessibility Considerations</h3>
+            <div className="pa">
+               Highlighted below are several accessibility features for adaptability and to ensure ease of access to the Checker functionality with keyboard or with a screen reader:
```
Comment (sharitrewin) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:967]:
```suggestion
               Highlighted below are several accessibility features for adaptability and to ensure ease of access to the Checker functionality, including with keyboard or with a screen reader: 
```

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx [hunk 2]
```diff
@@ -957,6 +962,94 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
               </ol>
             </div>
 
+            <h3 id="a11y_considerations">6.5 Accessibility Considerations</h3>
+            <div className="pa">
+               Highlighted below are several accessibility features for adaptability and to ensure ease of access to the Checker functionality with keyboard or with a screen reader: 
+              <ol
+                style={{ listStyleType: "decimal", marginInlineStart: "2rem" }}
+              >
+                <li>The Accessibility Checker tool is responsive to user's preferred font size and colors.
```
Comment (sharitrewin) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:971]:
```suggestion
                <li>The Accessibility Checker tool is responsive to the user's preferred font size and colors.
```

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx [hunk 3]
```diff
@@ -957,6 +962,94 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
...
+                  >
+                    <li>
+                      <p style={{ marginTop: "0rem" }}>
+                        Use the <strong>'tab'</strong> key to navigate to any focusable element in the checker, starting
+                        with the <strong>'Scan'</strong> button once the checker launches.
+                      </p>
+                    </li>
+                    <li>
+                      <p style={{ marginTop: "0rem" }}>
+                        After running the scan, press the <strong>'tab'</strong> key again, to navigate to the <strong>Reset Selections</strong> icon and the <strong>Report</strong> icon.
```
Comment (sharitrewin) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:989]: Earlier, the scan command was written as 'Scan', but here the Reset Selections and Report have no quotes.

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx [hunk 4]
```diff
@@ -957,6 +962,94 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
...
+                      </p>
+                    </li>
+                    <li>
+                      <p style={{ marginTop: "0rem" }}>
+                        Press the <strong>'tab'</strong> key to navigate to the checkbox by each issue type and press the <strong>'Enter'</strong> key to filter the list of issue by <strong>Violations</strong>, <strong>Needs review</strong> and/or by <strong>Recommendations</strong>.
```
Comment (sharitrewin) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:994]: We are using a small 't' for 'tab' but uppercase 'E' for 'Enter'.  Does our style guide dictate how to write keystrokes? We should be consistent with that.

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx [hunk 5]
```diff
@@ -957,6 +962,94 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
...
+                      </p>
+                    </li>
+                    <li>
+                      <p style={{ marginTop: "0rem" }}>
+                        Press the <strong>'tab'</strong> key to navigate to the <strong>Issue List</strong> tabs and use the <strong>'right'</strong> or the <strong>'left'</strong> arrow keys to navigate between the <strong>Element Roles</strong> view, the <strong>Checkpoints</strong> view and the <strong>Rules</strong> view.
```
Comment (sharitrewin) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:999]: Again, should Checklist, Element roles and Rules have quotes around them to be consistent with 'Scan'?

Comment (sharitrewin) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:999]: I think the arrow keys would be written 'left arrow' and 'right arrow'

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx [hunk 6]
```diff
@@ -957,6 +962,94 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
...
+                      </p>
+                    </li>
+                    <li>
+                      <p style={{ marginTop: "0rem" }}>
+                        Press the <strong>'tab'</strong> key to navigate to the <strong>issue grouping</strong> and use the <strong>'enter'</strong> to open or close the issue grouping. Press the <strong>'tab'</strong> key to navigate to each issue, and press the <strong>'enter'</strong> key to select any issue.
```
Comment (sharitrewin) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:1004]:
```suggestion
                        Press the <strong>'tab'</strong> key to navigate through the <strong>issue groupings</strong> associated with each checkpoint, element role or rule. Use the <strong>'enter'</strong> key to open or close an issue grouping. Within an open grouping, press the <strong>'tab'</strong> key to navigate to each issue, and press the <strong>'enter'</strong> key to select the current issue.
```

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx [hunk 7]
```diff
@@ -957,6 +962,94 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
...
+                      </p>
+                    </li>
+                    <li>
+                      <p style={{ marginTop: "0rem" }}>
+                        Press the <strong>'tab'</strong> key to move to the <strong>Learn more</strong> link or to move to the next issue.
```
Comment (sharitrewin) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:1009]: Quotes for 'Learn more'?

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx [hunk 8]
```diff
@@ -957,6 +962,94 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
...
+                      </p>
+                    </li>
+                  </ul>  
+                <li>
+                  Use the <strong>'headings'</strong> hierarchy or the implemented <strong>'landmarks'</strong> to quickly navigate from one section to another. The list of implemented landmarks are as follows:
```
Comment (sharitrewin) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:1014]: Headings and landmarks are not part of the user interface text, so shouldn't have quotes.

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx [hunk 9]
```diff
@@ -957,6 +962,94 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
...
+                </li>
+                  <ul
+                    style={{
+                      listStyleType: "circle",
+                      marginInlineStart: "2rem",
+                    }}
+                  >
+                    <li>
+                      <p style={{ marginTop: "0rem" }}>
+                        The <strong>IBM Accessibility Assessment</strong> or the <strong>IBM Accessibility Checker</strong>, main landmark: contains the main functionality of the tool in each view and includes,
```
Comment (sharitrewin) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:1024]: The comma before 'main landmark' and in the following list items, followed by a colon, makes the item a bit hard to parse. I suggest dropping those commas in all the list items.

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx [hunk 10]
```diff
@@ -957,6 +962,94 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
...
+                          }}
+                        >
+                          <li>
+                            <p style={{ marginTop: "0rem" }}>
+                              The <strong>Issue Count</strong>, region: contains the issue count by issue type as well as the total number of issues found.
+                            </p>
+                          </li>
+                          <li>
+                            <p style={{ marginTop: "0rem" }}>
+                              The <strong>Issue List</strong>, region: contains the list of issue grouped by Element Roles, Checkpoints or by Rules.
```
Comment (sharitrewin) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:1039]:
```suggestion
                              The <strong>Issue List</strong>, region: contains the list of issues grouped by Element Roles, by Checkpoints or by Rules.
```

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx [hunk 11]
```diff
@@ -957,6 +962,94 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
...
+                            </p>
+                          </li>
+                        </ul>
+                    </li>
+                    <li>
+                      <p style={{ marginTop: "0rem" }}>
+                        The <strong>Scan Summary</strong> or the <strong>Issue help</strong>, aside or the complementary landmark: in the Accessibility Assessment view, contains the scan summary, after the scan completes or shows the issue help when any issue is selected.
```
Comment (sharitrewin) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:1046]: I'm finding this unclear when written as one item.  Will users perceive these as a single aside or as two different ones?  If different, perhaps separate into two items.

```suggestion
                        The Accessibility Assessment view <strong>Scan Summary</strong> aside: contains the scan summary, after the scan completes.
                        
             The Accessibility Assessment view <strong>Issue Help</strong> aside: contains the issue help when any issue is selected.
```

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx (hunk 4 above)
Comment (cpandhi) [reply accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:994]: Missed that one, using lower case for all the keystrokes

**Comment by cpandhi** [comment]
All the conversation has been resolved, ready for approval.

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx [hunk 12]
```diff
@@ -986,32 +986,32 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
...
+                        Press the <strong>'tab'</strong> key to navigate to the checkbox by each issue type and press the <strong>'enter'</strong> key to filter the list of issue by <strong>Violations</strong>, <strong>Needs review</strong> and/or by <strong>Recommendations</strong>.
                       </p>
                     </li>
                     <li>
                       <p style={{ marginTop: "0rem" }}>
-                        Press the <strong>'tab'</strong> key to navigate to the <strong>Issue List</strong> tabs and use the <strong>'right'</strong> or the <strong>'left'</strong> arrow keys to navigate between the <strong>Element Roles</strong> view, the <strong>Checkpoints</strong> view and the <strong>Rules</strong> view.
+                        Press the <strong>'tab'</strong> key to navigate to the <strong>Issue List</strong> tabs and use the <strong>'right'</strong> or the <strong>'left'</strong> arrow keys to navigate between the <strong>'Element Roles'</strong> view, the <strong>'Checkpoints'</strong> view and the <strong>'Rules'</strong> view.
```
Comment (sharitrewin) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:999]:
```suggestion
                        Press the <strong>'tab'</strong> key to navigate to the <strong>Issue List</strong> tabs and use the <strong>'right arrow'</strong> or the <strong>'left arrow'</strong> keys to navigate between the <strong>'Element Roles'</strong> view, the <strong>'Checkpoints'</strong> view and the <strong>'Rules'</strong> view.
```

**Comment by cpandhi** [comment]
@eirckr ready for QA - issue 1528 https://github.ibm.com/IBMa/E2E/issues/1528
